test/runner.html

```diff
@@ -56,10 +56,8 @@
       'unit/styling-remote.html',
       'unit/styling-cross-scope-var.html',
       'unit/styling-cross-scope-apply.html',
-      // TODO(sorvell): disable until this Chrome bug is addressed:
-      // https://code.google.com/p/chromium/issues/detail?id=525280
-      // 'unit/styling-cross-scope-var.html?dom=shadow',
-      // 'unit/styling-cross-scope-apply.html?dom=shadow',
+      'unit/styling-cross-scope-var.html?dom=shadow',
+      'unit/styling-cross-scope-apply.html?dom=shadow',
```

Comment:
The related Chrome bug was apparently fixed in M46, so I have reactivated these.